fili-core/src/test/groovy/com/yahoo/bard/webservice/data/config/metric/makers/RawAggregationMetricMakerImplsSpec.groovy

```diff
@@ -0,0 +1,81 @@
+package com.yahoo.bard.webservice.data.config.metric.makers
+
+import com.yahoo.bard.webservice.data.metric.LogicalMetric
+import com.yahoo.bard.webservice.data.metric.MetricDictionary
+import com.yahoo.bard.webservice.data.metric.TemplateDruidQuery
+import com.yahoo.bard.webservice.data.metric.mappers.SketchRoundUpMapper
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.DoubleMaxAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.DoubleMinAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.DoubleSumAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.LongMaxAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.LongMinAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.LongSumAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.SketchCountAggregation
+import com.yahoo.bard.webservice.druid.model.aggregation.ThetaSketchAggregation
+
+import spock.lang.Specification
+import spock.lang.Unroll
+
+/**
+ *
+ * Created by mclawhor on 12/19/16.
+ */
+class RawAggregationMetricMakerImplsSpec extends Specification {
+
+    public static String NAME = "FOO"
+    public static String FIELD_NAME = "BAR"
+
+    @Unroll
+    def "Expected numeric aggregation is produced for #makerClass.getSimpleName()"() {
```

Comment:
I don't think the call to `getSimpleName()` is needed, since I believe Groovy defaults to that. That said, I also don't think method calls will work correctly with unrolling, only property-style accessors, so this should be `#makerClass.simpleName`

(Applies to other test as well)